src/scala/microsoft-spark-2.3.x/src/main/scala/org/apache/spark/deploy/DotnetRunner.scala

```diff
@@ -200,17 +195,6 @@ object DotnetRunner extends Logging {
     resolvedExecutable
   }
 
-  // When executing in YARN cluster mode, the name of the
```

Comment:
Note that for YARN cluster mode, the user has to ship the executable with dependencies via Zip file and the full path for the executable is constructed from it. Thus, this function is not needed.